index.css

```diff
@@ -1,361 +1,179 @@
-*{
+/* Reset và thiết lập chung */
+* {
     margin: 0;
     padding: 0;
-    
-}
-body{
+  }
+  
+  body {
     font-family: Arial, Helvetica, sans-serif;
     background-color: #f3f2f1;
-}
-/*top*/
-
-.top img{
+  }
+  
+  /* Biến CSS cho màu sắc và giá trị chung */
+  :root {
+    --primary-color: #25ac45;
+    --secondary-color: #eeda5f;
+    --text-color: black;
+    --border-color: orange;
+    --radius-large: 30px;
+  }
+  
+  /* Top */
+  .top img {
     width: 100%;
-    margin-bottom: 0;
-}
-
-/*menu*/
-.menu{
+  }
+  
+  /* Menu */
+  .menu {
     width: 100%;
-
     background-color: #464646;
-    margin-top: 0;
     text-align: center;
     line-height: 66px;
-}
-.menu ul li{
+    position: sticky;
+    top: 0;
+  }
+  
+  .menu ul li {
     display: inline;
-}
-.menu a{
+  }
+  
+  .menu a {
     color: white;
     text-decoration: none;
     padding: 13px 15px;
-}
-.menu {
-    position: sticky;
-    top:0;
-}
-
-/*menu-item*/
-.menu-item1 a{
-    color: #eeda5f;
+  }
+  
+  .menu-item1 a {
+    color: var(--secondary-color);
     font-size: 16px;
-}
-.menu-item2 :hover{
-    color: #eeda5f;
-    
-    
-}
-.menu-item2{
+  }
+  
+  .menu-item2:hover {
+    color: var(--secondary-color);
+  }
+  
+  .menu-item2, 
+  .menu-item1 {
     font-size: 16px;
-    
-}
-.menu-item3 a{
-color: #2cd655;
-font-size: 18px;
-
-}
-/*layout*/
-.grid-container {
-    display: grid;
-    grid-template-columns: auto auto auto ;
-    grid-gap:10px;
-    background-color:white;
-    padding-top: 30px;
-    padding-left: 15px;
   }

```

Comment:
Redundant font-size declaration. Merge these selectors to avoid repetition and improve maintainability.

```diff
- .menu-item2, 
- .menu-item1 {
+ .menu-item1, 
+ .menu-item2 {
    font-size: 16px;
  }
```